.github/workflows/workflow.yml

```diff
@@ -33,12 +33,13 @@ jobs:
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
           dune-cache: ${{ matrix.os == 'ubuntu-latest' }}
+          opam-depext: ${{ matrix.os == 'macos-11' }}
```

Comment:
```suggestion
          opam-depext: true
```

Reply:
If you enable it in all environments, you can remove the field. It's enabled by default.

Reply:
Yeah. I thought it was disabled by default, we will just remove the line it was only a test